### PR TITLE
Add all features for peer review

### PR DESCRIPTION
**Pull request by benshidanny11**
- Added all features(Display all tasks, Add task, Update task, and Delete task)

### CODE REVIEW

File: src/utils.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
+/* eslint-disable import/prefer-default-export */
+const todos = JSON.parse(localStorage.getItem('todos')) || [];
+export const addTodo = (description) => {
+  const index = todos && todos.length > 0 ? todos.length + 1 : 1;
+  const todo = { description, index, completed: false };
+  todos.push(todo);
+  localStorage.setItem('todos', JSON.stringify(todos));
```
Comment (Gambit142) [inline src/utils.js:7]: Hi @benshidanny11,

Great Work implementing the CRUD function, I like the fact that you follow best practices for HTML, CSS, and JavaScript.

In this code snippet, you repeated the statement that stores your objects in the local storage and the `window.location.reload()` function. Part of JavaScript practices is to try and make your code as **_DRY_** as possible.
So I will suggest that you put it in a function and call it in when you need it.

I hope to collaborate with you in the future. Happy Coding 👍

File: src/utils.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  window.location.reload();
```
Comment (Gambit142) [inline src/utils.js:14]: Hi @benshidanny11,

Great Work implementing the CRUD function, I like the fact that you follow best practices for HTML, CSS, and JavaScript.

In this code snippet, you repeated the statement that stores your objects in the local storage and the `window.location.reload()` function. Part of JavaScript practices is to try and make your code as **_DRY_** as possible.
So I will suggest that you put it in a function and call it in when you need it.

I hope to collaborate with you in the future. Happy Coding 👍

File: src/utils.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+export const deleteTodo = (index) => {
+  const remainingTodos = todos.filter((todo) => todo.index !== index);
+  if (remainingTodos.length > 0) {
+    remainingTodos.forEach((todo, index) => {
+      todo.index = index + 1;
+    });
+    localStorage.setItem('todos', JSON.stringify(remainingTodos));
+    window.location.reload();
+  } else {
+    localStorage.setItem('todos', JSON.stringify(remainingTodos));
```
Comment (Gambit142) [inline src/utils.js]: Hi, Please check the comments above

File: src/utils.js [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+  }
+};
+
+export const clearAllComplele = () => {
+  const remainingTodos = todos.filter((todo) => !todo.completed);
+  if (remainingTodos.length > 0) {
+    remainingTodos.forEach((todo, index) => {
+      todo.index = index + 1;
+    });
+    localStorage.setItem('todos', JSON.stringify(remainingTodos));
```
Comment (Gambit142) [inline src/utils.js]: Please check the first comment

**Review by Gambit142 — CHANGES_REQUESTED** [review]
## Changes Required ♻️ 

Great job!! @benshidanny11, 

Your work is well documented and you were professional in creating it.

However, there are few changes to be made. Kindly check the comments below to see where necessary changes can be made.

Please do reach out to me if you have any questions or if you need further clarifications.

**Comment by benshidanny11** [comment]
Thank you @Gambit142, I really appreciate your feedback and I am going to work on them :+1:

**Review by Gambit142 — APPROVED** [review]
## PROJECT APPROVED ✔️ 

Hi @benshidanny11 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Happy Coding 👏 👏

**Pull request merged**